packages/node_modules/@ciscospark/widget-recents/src/selector.js

```diff
@@ -211,6 +211,19 @@ const getRecentSpacesWithDetail = createSelector(
   }
 );
 
+const getCurrentUserWithAvatar = createSelector(
+  [getCurrentUser, getAvatars], (currentUser, avatars) => {
+    let user;
+
+    if (currentUser && currentUser.id) {
+      // user = Object.assign(currentUser, {img: avatars.getIn(['items', currentUser.id])});
+      user = Object.assign({}, currentUser.toJS(), {img: avatars.getIn(['items', currentUser.id])});
```

Comment:
Is this working now?

Reply:
no. it does not also work on Netlify

Reply:
I'm curious, on how you made it work on your machine. did you open the recent widget? thats when it crashes.